stories/plugins/tables.stories.tsx

```diff
@@ -36,6 +57,15 @@ export const Example = () => {
         value={value}
         onChange={newValue => setValue(newValue)}
       >
+        <HeadingToolbar>
+          <ToolbarMark format={MARK_BOLD} icon={<FormatBold />} />
+          <ToolbarTable action={insertTable} icon={<BorderAll />} />
+          <ToolbarTable action={deleteTable} icon={<BorderClear />} />
```

Comment:
In the future we can add a floating delete button.

[Confluence example](https://confluence.atlassian.com/confcloud/add-formatting-to-your-page-724764836.html#Addformattingtoyourpage-tablesTables):
![image](https://user-images.githubusercontent.com/19695832/73827374-b78e2480-47ff-11ea-8d98-f24ea5901b48.png)

GitBook example (in the sidebar):
![image](https://user-images.githubusercontent.com/19695832/73826939-f4a5e700-47fe-11ea-81d5-bcd4ae4c9545.png)

Reply:
We can use quill-icons later https://canner.github.io/canner-slate-editor/?selectedKind=Quill%20icons&selectedStory=List&full=0&addons=1&stories=1&panelRight=0